lib/simulator-xcode-6.js

```diff
@@ -100,16 +100,10 @@ class SimulatorXcode6 extends EventEmitter {
    * or, for iOS 7.1, the app name without `.app` (e.g., MobileSafari)
    */
   async getAppDir (id, subDir = 'Data') {
-    if (await this.isFresh()) {
-      log.info('Attempted to get an app path from a fresh simulator ' +
-               'quickly launching the sim to populate its directories');
-      await this.launchAndQuit();
```

Comment:
When updating settings, etc., we manually use `isFresh` and `launchAndQuit`. See, for instance, https://github.com/appium/appium-ios-driver/blob/5393545c51f983873e34b5919e78ab1ad25cfbcc/lib/settings.js#L39